Cpp2IL.Core/OutputFormats/AsmResolverDllOutputFormatDefault.cs

```diff
@@ -0,0 +1,17 @@
+using AsmResolver.DotNet;
+using AssetRipper.CIL;
+using Cpp2IL.Core.Model.Contexts;
+
+namespace Cpp2IL.Core.OutputFormats;
+
+public class AsmResolverDllOutputFormatDefault : AsmResolverDllOutputFormat
+{
+    public override string OutputFormatId => "dll_default";
```

Comment:
Can we rename dll_default to dummydll, because this is a breaking change in the command line interface for e.g. melonloader